clang/test/lab1/simonyan_suren/demo.cpp

```diff
@@ -0,0 +1,58 @@
+// RUN: %clang_cc1 -load %llvmshlibdir/AlwaysInlinePlugin%pluginext -plugin always-inlines-plugin %s 2>&1 | FileCheck %s
+
+// CHECK: Added attribute always_inline in sum
+int sum(int A, int B) { return A + B; }
```

Comment:
Could you please check that the attribute is added into AST tree?

Reply:
Add the attribute to AST tree check done